email_validator/validate_email.py

```diff
@@ -4,17 +4,17 @@
 
 
 def validate_email(
-    email,
+    email: str | bytes,
     # /, # not supported in Python 3.6, 3.7
     *,
-    allow_smtputf8=None,
-    allow_empty_local=False,
-    check_deliverability=None,
-    test_environment=None,
-    globally_deliverable=None,
-    timeout=None,
-    dns_resolver=None
-):
+    allow_smtputf8: bool | None = None,
```

Comment:
I love this syntax with `|`, but unfortunately it only came about with Python 3.10. If you want to support just 3.10 and newer, it's definitely the way to go. Otherwise, you have to do either:

```
from typing import Union
allow_smtputf8: Union[bool, None] = None
```

Or:

```
from typing import Optional
allow_smtputf8: Optional[bool] = None
```

Reply:
Thanks! Fixing.